lib/app_annie/meta_data.rb

```diff
@@ -1,6 +1,6 @@
 module AppAnnie
   class MetaData
-    API_ROOT = "/v1.2/".freeze
+    API_ROOT = "/v1.2".freeze
```

Comment:
Where'd we catch this and what's it address?

Reply:
Oh, I forgot about this. A spec is failing because this makes a request with doubled up `//`. I'll update the commit message.

```
     Faraday::Adapter::Test::Stubs::NotFound:
       no stubbed request for get /v1.2//apps/google-play/package-codes2ids?package_codes=com.example.app 
```

Reply:
:thinking: ... :man_technologist: ...  :+1: